src/ansys/fluent/core/scheduler/load_machines.py

```diff
@@ -118,21 +118,18 @@ def _parse_host_info(host_info):
         A list of dictionaries formatted as:
         {'machine-name' : ###, 'core-count' : ###}
     """
-    if (
-        (":" in host_info or "," in host_info)
-        and not "\\" in host_info
-        and not "/" in host_info
-    ):
-        # Filenames generally shouldn't have ':',
-        # so assume it's a string list and parse accordingly
-        sMod = 1 if host_info[0] == "[" else 0
-        sBeg = sMod
-        sEnd = len(host_info) - sMod
-        machine_data = host_info[sBeg:sEnd].split(",")
-    else:
+    print(host_info)
+    if "\\" in host_info or "/" in host_info or "." in host_info:
```

Comment:
I think [Path.is_file()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file) can be used here

Reply:
I chose to use os.path.exists(host_info) as OS is already inported. Functioning in this way I removed the test_file_not_found. Would it make sense to add the test back or something similar?

Reply:
@Adam-Boutin : I agree with @mkundu1 that the more explicit query is preferable.

Reply:
@seanpearsonuk I am unsure of what you mean. Would you be able to give me some more info?

Reply:
It's the suggestion made by mkundu1.  You can replace the entire logic with a call to `Path.is_file()` because it's just trying to work out if `host_info` is a file or not.   I think you have to construct the `Path` object to do that.

Reply:
What would be the difference between using Path(host_info).is_file() vs. os.path.exists(host_info)?

Reply:
I switched it over to `Path(host_info).is_file()`. I saw in "test_scheduler.py" `os.path.isfile()` do each of these methods have a specific use case or would both `.is_file()` and `.isfile()` be interchangeable? 

Is there much of a downside to importing more modules?

Reply:
Both `Path.is_file()` and `os.path.isfile()` are interchangeable ([doc](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module)). Using the Path module is the modern usage, so that will be preferred in general.

Reply:
> I switched it over to `Path(host_info).is_file()`. I saw in "test_scheduler.py" `os.path.isfile()` do each of these methods have a specific use case or would both `.is_file()` and `.isfile()` be interchangeable?
> 
> Is there much of a downside to importing more modules?

There's not really a downside to importing more modules.  As mkundu1 says though, the `PathLib` module is the preferred modern alternative to `os.path`